assembl/static2/js/app/actions/adminActions/index.js

```diff
@@ -5,8 +5,4 @@ export const updateEditLocale = newLocale => ({
 
 export const addLanguagePreference = locale => ({ locale: locale, selected: true, type: 'ADD_LANGUAGE_PREFERENCE' });
 
-export const removeLanguagePreference = locale => ({ locale: locale, selected: false, type: 'REMOVE_LANGUAGE_PREFERENCE' });
-
-export const languagePreferencesHasChanged = state => ({ state: state, type: 'LANGUAGE_PREFERENCE_HAS_CHANGED' });
-
```

Comment:
addLanguagePreference shouldn't be removed as well?

Reply:
C'est utilisé par le menu des langues à droite dans l'admin

Reply:
Oui ça doit être supprimé de js/app/components/navbar/languageMenu.jsx justement, voir le commentaire qu'il y a là-bas. C'était un effet de bord pour faire fonctionner l'admin.

Reply:
done in https://github.com/assembl/assembl/commit/3500307c382a80f5dfdf90794ad14c747afdcf32